drivers/base/regmap/regmap-sdw-mbq.c

```diff
@@ -0,0 +1,100 @@
+// SPDX-License-Identifier: GPL-2.0
+// Copyright(c) 2020 Intel Corporation.
+
+#include <linux/device.h>
+#include <linux/mod_devicetable.h>
+#include <linux/module.h>
```

Comment:
```
#include <linux/errno.h>
#include <linux/regmap.h>
```

Reply:
ok, will also add in the regular SoundWire code then. Thanks for spotting this

Reply:
fixed